{
  "candidate": "CVE-2008-0983",
  "description": "lighttpd 1.4.18, and possibly other versions before 1.5.0, does not properly calculate the size of a file descriptor array, which allows remote attackers to cause a denial of service (crash) via a large number of connections, which triggers an out-of-bounds access.",
  "notes": {
    "jdstrand": "per emgent, this is fixed with 90_maxfds_crash_fix"
  },
  "priority": "low",
  "bugs": [
    "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=466663",
    "https://bugs.launchpad.net/ubuntu/+bug/195380"
  ],
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-0983"
  ],
  "public_date": "2008-02-26 18:44:00 UTC",
  "packages": {
    "dapper": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "released",
        "note": "1.4.11-3ubuntu3.6"
      }
    },
    "devel": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "released",
        "note": "1.4.18-1ubuntu3"
      }
    },
    "edgy": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "released",
        "note": "1.4.13~r1370-1ubuntu1.4"
      }
    },
    "feisty": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "released",
        "note": "1.4.13-9ubuntu4.3"
      }
    },
    "gutsy": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "released",
        "note": "1.4.18-1ubuntu1.1"
      }
    },
    "upstream": {
      "lighttpd": {
        "name": "lighttpd",
        "status": "needed"
      }
    }
  }
}
